### PR TITLE
Continuous assignment simulation

### LINKED ISSUE #540: Continuous (non-grouped) assignments
We need to simulate not only the grouped assignments but also continuous assignments

- [x] Incorporate continuous assignments into group simulation
- [x] Running lowered futil (no control)
- [x] Add more tests

### PR DESCRIPTION
**Pull request by yn224**
Closes #540 

Enabling continuous assignment simulation for the lowered versions of Calyx programs. There are some test cases that still needs to be covered as detailed in the issue #574.

### CODE REVIEW

**Comment by EclecticGriffin** [comment]
Just a note to resolve the conflicts with the main branch. Lmk when you're ready for a more thorough review

File: interp/src/interpreter/interpret_group.rs [hunk 1]
```diff
@@ -135,13 +135,35 @@ impl WorkingEnvironment {
...
+    //     {
+    //         &Primitive::StdReg(ref reg) => {
+    //             println!("  internal state: {}", reg.val)
+    //         }
+    //         &Primitive::StdMemD1(ref mem) => {
+    //             println!("  memval : {}", mem.data[0])
+    //         }
+    //         _ => {}
+    //     }
+    // }
```
Comment (EclecticGriffin) [inline interp/src/interpreter/interpret_group.rs:160]: We should probably either commit to leaving the debugging code or remove the comment

**Review by EclecticGriffin — COMMENTED** [review]
Looks good in general, though I confess that I've trouble reading through large calyx blobs, so it's a very possible that I missed something.

Maybe @rachitnigam has some insight on this, but it would be nice if we could use runt to lower the existing tests, rather than committing a bunch of lowered calyx, especially since that will make us aware of any breaking changes.

I think this can be done by using the `expect_dir` key in runt and then making the lowered expects have the same name as the base file but in a different folder to avoid collisions.

**Comment by rachitnigam** [comment]
Reorganized test suite in #586.

**Review by rachitnigam — APPROVED** [review]
Fix the comments mentioned by @EclecticGriffin and merge. Once you're done, I'll merge #586 with the appropriate test suite updates.

**Pull request merged**